pkg/chrootarchive/archive_unix.go

```diff
@@ -72,24 +108,12 @@ func invokeUnpack(decompressedArchive io.Reader, dest string, options *archive.T
 		return fmt.Errorf("untar pipe failure: %w", err)
 	}
 
-	if root != "" {
-		relDest, err := filepath.Rel(root, dest)
-		if err != nil {
-			return err
-		}
-		if relDest == "." {
-			relDest = "/"
-		}
-		if relDest[0] != '/' {
-			relDest = "/" + relDest
-		}
-		dest = relDest
-	}
-
-	cmd := reexec.Command("storage-untar", dest, root)
+	cmd := reexec.Command("storage-untar", dest, "/proc/self/fd/4")
 	cmd.Stdin = decompressedArchive
 
-	cmd.ExtraFiles = append(cmd.ExtraFiles, r)
+	cmd.ExtraFiles = append(cmd.ExtraFiles, r) // fd 3
+	// If you change this, change rootFileDescriptor above too
```

Comment:
This is nice, but then `"/proc/self/fd/4"` is independently hard-coded in two places. Make that a constant as well

(Alternatively, we could turn `procPathForFd` into a repo-wide internal helper somewhere. That seems out of scope for this PR. Or, maybe, we could just drop the parameter and hard-code the behavior, but that assumes the caller and the reexec destination can _never_ get out of sync… probably a bit risky for CRI-O.)